src/internal/connector/discovery/discovery.go

```diff
@@ -0,0 +1,83 @@
+package discovery
+
+import (
+	"context"
+
+	msgraphgocore "github.com/microsoftgraph/msgraph-sdk-go-core"
+	"github.com/microsoftgraph/msgraph-sdk-go/models"
+	msuser "github.com/microsoftgraph/msgraph-sdk-go/users"
+	"github.com/pkg/errors"
+
+	"github.com/alcionai/corso/src/internal/connector/graph"
+	"github.com/alcionai/corso/src/internal/connector/support"
+)
+
+const (
+	userSelectID            = "id"
+	userSelectPrincipalName = "userPrincipalName"
+	userSelectDisplayName   = "displayName"
+)
+
+func Users(ctx context.Context, gs graph.Service, tenantID string) ([]models.Userable, error) {
+	users := make([]models.Userable, 0)
+
+	options := &msuser.UsersRequestBuilderGetRequestConfiguration{
+		QueryParameters: &msuser.UsersRequestBuilderGetQueryParameters{
+			Select: []string{userSelectID, userSelectPrincipalName, userSelectDisplayName},
+		},
```

Comment:
Use of the call in query_options would ensure the options are tested or if there is a change involving the use of options  in the future. 

```go
opts, err := optionsForUsers(userSelectPrincipalName, userSelectDisplayName)
```

Reply:
@dadams39 - it wasn't clear to me how that test was helping add any more coverage. What are we testing for?

Reply:
Discussed offline. `optionsForUsers` used to be under the exchange pkg. This `Users` call pulls out that logic into `connector/discovery` which can be used as a regression test for Graph changes.